packages/@sanity/code-input/src/config.ts

```diff
@@ -1,21 +1,30 @@
 import {CodeInputLanguage} from './types'
 
+// NOTE: MAKE SURE THESE ALIGN WITH IMPORTS IN ./editorSupport
 export const SUPPORTED_LANGUAGES: CodeInputLanguage[] = [
   {title: 'Batch file', value: 'batchfile'},
+  {title: 'C#', value: 'csharp'},
   {title: 'CSS', value: 'css'},
-  {title: 'SASS', value: 'sass'},
-  {title: 'SCSS', value: 'scss'},
+  {title: 'Go', value: 'golang'},
+  {title: 'GROQ', value: 'groq'},
   {title: 'HTML', value: 'html'},
+  {title: 'Java', value: 'java'},
   {title: 'JavaScript', value: 'javascript'},
-  {title: 'TypeScript', value: 'typescript'},
   {title: 'JSON', value: 'json'},
   {title: 'JSX', value: 'jsx'},
   {title: 'Markdown', value: 'markdown'},
+  {title: 'MySQL', value: 'mysql'},
   {title: 'PHP', value: 'php'},
+  {title: 'Plain text', value: 'text'},
   {title: 'Python', value: 'python'},
+  {title: 'Ruby', value: 'ruby'},
+  {title: 'SASS', value: 'sass'},
+  {title: 'SCSS', value: 'scss'},
   {title: 'sh', value: 'sh'},
-  {title: 'Plain text', value: 'text'},
-  {title: 'GROQ', value: 'groq'},
+  {title: 'TSX', value: 'tsx'},
+  {title: 'TypeScript', value: 'typescript'},
+  {title: 'XML', value: 'xml'},
+  {title: 'Yarn', value: 'yarn'},
```

Comment:
oh I see, should this be `yaml` not `yarn`?

Reply:
Most def should!